File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        const hashSet = new Set(nums)
        let maxi = 0, result = 0
        for(let num of nums){
            if(!hashSet.has(num-1)){
                while(hashSet.has(num+maxi)){
                    maxi++
                }
                result = Math.max(result,maxi)
            }
            maxi=0
        }
        return result
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-2.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        i = 1
        k = 1
        while i<len(nums):
            if nums[i] != nums[k-1]:
                nums[k] = nums[i]
                k+=1
            i+=1



        return k